Xamarin.Forms.Platform.iOS/CollectionView/SelectableItemsViewRenderer.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.ComponentModel;
+
+namespace Xamarin.Forms.Platform.iOS
+{
+	public class SelectableItemsViewRenderer : ItemsViewRenderer
+	{
+		SelectableItemsView SelectableItemsView => (SelectableItemsView)Element;
+		SelectableItemsViewController SelectableItemsViewController => (SelectableItemsViewController)ItemsViewController;
+
+		protected override ItemsViewController CreateController(ItemsView itemsView, ItemsViewLayout layout)
+		{
+			return new SelectableItemsViewController(itemsView as SelectableItemsView, layout);
+		}
+
+		protected override void OnElementPropertyChanged(object sender, PropertyChangedEventArgs changedProperty)
+		{
+			base.OnElementPropertyChanged(sender, changedProperty);
+
+			if (changedProperty.Is(SelectableItemsView.SelectedItemProperty))
+			{
+				UpdateNativeSelection();
+			}
+			else if (changedProperty.Is(SelectableItemsView.SelectionModeProperty))
+			{
+				SelectableItemsViewController.UpdateSelectionMode();
+			}
+		}
+
+		protected override void SetUpNewElement(ItemsView newElement)
+		{
+			if (newElement != null && !(newElement is SelectableItemsView))
+			{
+				throw new ArgumentException($"{nameof(newElement)} must be of type {typeof(SelectableItemsView).Name}");
+			}
```

Comment:
Not for this PR, but I think we need to create an extension method to do this:

`newElement.EnsureTypeOrNull<SelectableItemsView>()`

I know I have been using this chunk of code for material...